### PR TITLE
tdp-2427:changed gather docs

### PR DESCRIPTION
**Pull request by vsmith-bw**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `gitbook` 

- [ ] I made sure that the site looks great and functions perfectly live
- [ ] I ran splell check
- [ ] I'm ready for these changes to be made live

### General Review

- [ ] All changes/updates requested were intentionally changed or added (no extraneous file or partial updates)
- [ ] The live rendered page behaves and looks like intended.
- [ ] All links work as intended. (click every link to make sure that it takes the user to the expected place).
- [ ] The markdown was rendered to HTML as intended. (Tables look right, code samples are gated properly).
- [ ] Any style or CSS changes are tested on multiple different pages to ensure nothing unexpected broke or changed.
- [ ] I proof-read the live site (`gitbook serve`) and there are no oblivious splelling or grammer misteaks.

### Gitbook Specific

- [ ] Any new documentation pages have been added to the `SUMMARY.md` file so they are rendered to HTML.
- [ ] Any links to other pages within this documentation use relative links (`[read more about ...](guides/voice/voicemail.md)`).
- [ ] Gitbook can install and build the documentation (`gitbook install` & `gitbook build`)

### Code Specific

- [ ] All code has been tested against the live API.
- [ ] All code can be run without throwing errors or **new** warnings.
- [ ] All code is formatted correctly and consistently (spaces, tabs).
- [ ] All code is legible and idiomatic to the language.
- [ ] All code uses sensical method, function, and variable names.
- [ ] All code samples use the appropriate syntax highlighting.
- [ ] Any examples re-use same phone numbers, uuids, variable names, etc... throughout (IE don't have something like `{from: '+19191231234', to: '+19191231234'}` unless it's meant to show the same phone number calling/texting itself ).

## TL;DR

- [ ] I did these things
- [ ] I'm ready for these changes to be made live

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-tdp-2427.s3-website-us-east-1.amazonaws.com/

File: voice/bxml/verbs/gather.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ completion of the gather. BXML returned by that callback are then executed. If `
 If no `gatherUrl` attribute is specified, the gathered digits are discarded and execution of verbs following the `<Gather>` continues.
 
 ### Nestable Verbs
-The following verbs may be nested inside of a `<Gather>` tag.  A maximum of one verb is allowed:
+The following verbs may be nested inside of a `<Gather>` tag.  The number of verbs that can be nested inside a Gather is 20 / repeatCount.
```
Comment (n-miles) [inline voice/bxml/verbs/gather.md]: I would say something more like
```
The number of audio clips played inside a `<Gather>` tag may not exceed 20, including those resulting from the `repeatCount` attribute. The total number of audio clips is equal to the `repeatCount` times the number of nested verbs.
```

File: voice/bxml/verbs/gather.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ The Gather verb is used to collect digits for some period of time.
...
 | firstDigitTimeout | (optional) Time (in seconds) to pause after any audio from nested `<SpeakSentence>` or `<PlayAudio>` verb is played (in seconds) before terminating the Gather. Default value is 5. Range: decimal values between 0 - 60.                                                                                                           |
-| repeatCount       | (optional) The number of times the audio prompt should be repeated if no digits are pressed. For example, if this value is `3`, the nested audio clip will be played a maximum of three times. The delay between repetitions will be equal to `firstDigitTimeout`. Default value is 1. Range: 1-25.|
+| repeatCount       | (optional) The number of times the audio prompt should be repeated if no digits are pressed. For example, if this value is `3`, the nested audio clip will be played a maximum of three times. The delay between repetitions will be equal to `firstDigitTimeout`. Default value is 1. Range: 1-20.|
```
Comment (nashley-bw) [inline voice/bxml/verbs/gather.md]:
```suggestion
| repeatCount       | (optional) The number of times the audio prompt should be played if no digits are pressed. For example, if this value is `3`, the nested audio clip will be played a maximum of three times. The delay between repetitions will be equal to `firstDigitTimeout`. Default value is 1. Range: 1 to 20.|
```

Comment (nashley-bw) [reply voice/bxml/verbs/gather.md]: I think `The number of times the audio prompt should be repeated` is misleading.

File: voice/bxml/verbs/gather.md (hunk 1 above)
Comment (nashley-bw) [inline voice/bxml/verbs/gather.md]:
```suggestion
The following verbs may be nested inside of a `<Gather>` tag.  The number of nested verbs times the `repeatCount` must not exceed 20.
```

File: voice/bxml/verbs/gather.md (hunk 2 above)
Comment (nashley-bw) [reply voice/bxml/verbs/gather.md]: Regarding `Range: x to y.`, I don't really care which format we use, but I think we should be consistent throughout our documentation, so whichever you choose, I think you should modify the other ranges to match.

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-tdp-2427.s3-website-us-east-1.amazonaws.com/

File: voice/bxml/verbs/gather.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ The Gather verb is used to collect digits for some period of time.
...
 | firstDigitTimeout | (optional) Time (in seconds) to pause after any audio from nested `<SpeakSentence>` or `<PlayAudio>` verb is played (in seconds) before terminating the Gather. Default value is 5. Range: decimal values between 0 - 60.                                                                                                           |
-| repeatCount       | (optional) The number of times the audio prompt should be repeated if no digits are pressed. For example, if this value is `3`, the nested audio clip will be played a maximum of three times. The delay between repetitions will be equal to `firstDigitTimeout`. Default value is 1. Range: 1-25.|
+| repeatCount       | (optional) The number of times the audio prompt should be played if no digits are pressed. For example, if this value is `3`, the nested audio clip will be played a maximum of three times. The delay between repetitions will be equal to `firstDigitTimeout`. Default value is 1. Range: 1-20.|
```
Comment (ghost) [inline voice/bxml/verbs/gather.md]: Ranged: 1 - 20

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-tdp-2427.s3-website-us-east-1.amazonaws.com/

File: voice/bxml/verbs/gather.md [hunk 4]
```diff
@@ -29,7 +29,7 @@ completion of the gather. BXML returned by that callback are then executed. If `
 If no `gatherUrl` attribute is specified, the gathered digits are discarded and execution of verbs following the `<Gather>` continues.
 
 ### Nestable Verbs
-The following verbs may be nested inside of a `<Gather>` tag.  A maximum of one verb is allowed:
+The following verbs may be nested inside of a `<Gather>` tag.  If using a repeat count of one, up to 20 verbs can be nested. Refer to repeatCount Attribute if using a value greater than 1.
```
Comment (ghost) [inline voice/bxml/verbs/gather.md]: put ticks around repeatCount and make attribute lowercase.

Comment (vsmith-bw) [reply voice/bxml/verbs/gather.md]: done

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-tdp-2427.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-tdp-2427.s3-website-us-east-1.amazonaws.com/

**Review by ghost — APPROVED** [review]

**Review by n-miles — APPROVED** [review]

File: voice/bxml/verbs/gather.md [hunk 5]
```diff
@@ -29,7 +29,7 @@ completion of the gather. BXML returned by that callback are then executed. If `
 If no `gatherUrl` attribute is specified, the gathered digits are discarded and execution of verbs following the `<Gather>` continues.
 
 ### Nestable Verbs
-The following verbs may be nested inside of a `<Gather>` tag.  A maximum of one verb is allowed:
+The following verbs may be nested inside of a `<Gather>` tag.  If using a repeat count of one, up to 20 verbs can be nested. Refer to `repeatCount` attribute if using a value greater than 1.
```
Comment (nashley-bw) [inline voice/bxml/verbs/gather.md:32]:
```suggestion
The following verbs may be nested inside of a `<Gather>` tag.  If using a repeat count of `1`, up to `20` verbs can be nested. Refer to `repeatCount` attribute if using a value greater than `1`.
```

File: voice/bxml/verbs/gather.md [hunk 6]
```diff
@@ -15,7 +15,7 @@ The Gather verb is used to collect digits for some period of time.
...
 | firstDigitTimeout | (optional) Time (in seconds) to pause after any audio from nested `<SpeakSentence>` or `<PlayAudio>` verb is played (in seconds) before terminating the Gather. Default value is 5. Range: decimal values between 0 - 60.                                                                                                           |
-| repeatCount       | (optional) The number of times the audio prompt should be repeated if no digits are pressed. For example, if this value is `3`, the nested audio clip will be played a maximum of three times. The delay between repetitions will be equal to `firstDigitTimeout`. Default value is 1. Range: 1-25.|
+| repeatCount       | (optional) The number of times the audio prompt should be played if no digits are pressed. For example, if this value is `3`, the nested audio clip will be played a maximum of three times. The delay between repetitions will be equal to `firstDigitTimeout`. Default value is 1. `repeatCount` * number of verbs must not be greater than 20|
```
Comment (nashley-bw) [inline voice/bxml/verbs/gather.md:18]:
```suggestion
| repeatCount       | (optional) The number of times the audio prompt should be played if no digits are pressed. For example, if this value is `3`, the nested audio clip will be played a maximum of three times. The delay between repetitions will be equal to `firstDigitTimeout`. Default value is `1`. `repeatCount` * number of verbs must not be greater than `20` |
```

**Pull request merged**